ash/src/entry.rs

```diff
@@ -163,11 +164,27 @@ impl<L> EntryV1_2 for EntryCustom<L> {
 }
 
 impl<L> EntryCustom<L> {
-    pub fn new_custom<Load>(mut lib: L, mut load: Load) -> Self
+    pub fn new_custom<Load>(
+        mut lib: L,
+        mut load: Load,
+    ) -> std::result::Result<Self, MissingEntryPoint>
     where
         Load: FnMut(&mut L, &::std::ffi::CStr) -> *const c_void,
     {
-        let static_fn = vk::StaticFn::load(|name| load(&mut lib, name));
+        // TODO: Make this a &'static CStr once CStr::from_bytes_with_nul_unchecked is const
+        static ENTRY_POINT: &'static [u8] = b"vkGetInstanceProcAddr\0";
+
+        // Bypass the normal StaticFn::load so we can return an error
+        let static_fn = vk::StaticFn {
```

Comment:
Perhaps cleaner to move this to an `impl vk::StaticFn` block in this file so that it can be used by external applications as well? `vk::StaticFn::load` is after all public as well.